README.md

```diff
@@ -1,104 +1,46 @@
-
-3. Multiple windows with other options of operators can be viewed. A maximum of 16 windows are supported by VisIt. The following with an input of plot lists and operator lists can be used:
-
-        python MakeImagesTerminal.py -pl [plot list] -op [operator list] -wi
-
-        python MakeImagesTerminal.py -pl "[['test.vtk','Pseudocolor','TALLY_TAG','Log',('Min',0.00001),('Max',0.0001)], ['test.vtk','Contour','ERROR_TAG','DASH'],]" -op "[{'Clip': {'oct': (1, 1, 1)}},{'Clip': {'oct': (-1, 1, 1)}},[{'Clip': {'oct': (1, 1, 1)}}, ['Slice', ['x', 10]]]]" -wi
-
-
-4. Multiple axis can be inserted without delimiters for automatic slicing. The number of slices across each axis can be defined. Slices are programmed to be evenly distributed between maximum and minimum spatial dimensions of Pseudocolor plots. The first two are of the maximum and minimum spatial dimensions. More than two slices are needed for this option to work. To invoke this function the following can be used:
-
-        python MakeImagesTerminal.py -pl [plot list] -mu [axes, number of slices]
-
-        python MakeImagesTerminal.py -pl "[['test.vtk','Pseudocolor','TALLY_TAG','Log',('Min',0.00001),('Max',0.0001)], ['test.vtk','Contour','ERROR_TAG','DASH'],]" -op "[[{'Clip': {'oct': (1, 1, 1)}},{'Clip': {'oct': (-1, 1, 1)}}]]" -mu "['xyz',2]"
-
-
-5. Multiple orbital views can be gathered horizontally, vertically, or both. The number of images for each orbit can be defined. The larger the number of images the "smoother" an animation would be from collected images. 
-
-        python MakeImagesTerminal.py -pl [plot list] -op [operator list] -or "['yes','view',number of views]"
-
-        python MakeImagesTerminal.py -pl "[['test.vtk','Pseudocolor','TALLY_TAG','Log',('Min',0.00001),('Max',0.0001)], ['test.vtk','Contour','ERROR_TAG','DASH'],]" -op "[{'Clip': {'oct': (1, 1, 1)}},{'Clip': {'oct': (-1, 1, 1)}}]" -or "['both',20]"
+A sample output from an h5m geometry file and a vtk data file can be seen below.
 
+<p align="center">
+<img src="https://i.postimg.cc/Fzb5jM6t/Screenshot-from-2018-11-02-13-37-55.png" width="400" height="400"/>
+</p>
 
 ----------------------------------------
 
-Bash scripts for removing a graveyard from an h5m file, extracting surfaces, extracting, curves, and substituting loaded datasets are under /BashTool.
-
-To convert an h5m file to stl or a meshtal file to vtk, the following examples can be used:
-
-        python MakeImagesTerminal.py -d1 "['meshtal','test.vtk']"
-        python MakeImagesTerminal.py -d2 "['fng_zip.h5m','test.stl']"  
-
-To remove a graveyard, the following example can be used:
-
-        python MakeImagesTerminal.py -gr "['fng_zip.h5m','rmgrave.stl']"
-
-To extract surfaces, the following example can be used:
+2. GraveIdentifyAndRemove.py can be used to remove the graveyard EntitySet from an h5m geometry file. The user may specify a specific output file name and extension. It can also extract curve, surface, and volume EntitySets.
```

Comment:
I'm not sure as to whether or not it's worth mentioning entity sets here (though what you've written is entirely correct). Perhaps "graveyard volume" would do? Thoughts @gonuke?